scd/scd.c

```diff
@@ -304,20 +304,20 @@ scd_logfile_rename_cb(UNUSED event_timer_t *timer, UNUSED void *data)
 	logf_handler_set_prio(scd_logfile_handler, LOGF_PRIO_TRACE);
 }
 
-int
-main(int argc, char **argv)
+static void INIT
+main_init(void)
 {
-	ASSERT(argc >= 1);
```

Comment:
This block is removed without substitution, right?

Reply:
nope this is still checked in main() see line 320. The diff is somehow misleading here.

Reply:
What about lines 312-315?

Reply:
correct these were android specific leftovers from initial trustme version for mobile devices